fdbserver/LogRouter.actor.cpp

```diff
@@ -33,6 +33,21 @@
 #include "flow/Stats.h"
 #include "flow/actorcompiler.h"  // This must be the last #include.
 
+struct LogRouterStats {
+	CounterCollection cc;
```

Comment:
Any reason to not just directly add this to LogRouterData?  Having this be a separate class that just reaches back into the members of LogRouterData via pointers feels a bit odd.